samples/rtos/dynamic-threads/main.cpp

```diff
@@ -47,12 +47,20 @@ typedef void (*EntryFunction)(ULONG thread_input);
 ThreadData dataArray[ThreadNum + 1];
 
 void ThreadCreator(TX_BYTE_POOL* byte_pool, TX_THREAD* thread, EntryFunction thread_entry, ULONG thread_input) {
-    IO::UART& uart = IO::getUART<IO::Pin::UART_TX, IO::Pin::UART_RX>(9600);
-    char* pointer = static_cast<char*>(TX_NULL);
+    io::UART& uart = io::getUART<io::Pin::UART_TX, io::Pin::UART_RX>(9600);
+    char* pointer  = static_cast<char*>(TX_NULL);
 
     tx_byte_allocate(byte_pool, (void**) &pointer, DEMO_STACK_SIZE, TX_NO_WAIT);
-    ULONG status = tx_thread_create(thread, "dynamic thread", thread_entry, thread_input, pointer,
-                                    DEMO_STACK_SIZE, 1, 1, TX_NO_TIME_SLICE, TX_AUTO_START);
+    ULONG status = tx_thread_create(thread,
```

Comment:
I still don't like this, but I'm not sure there's anything to be done about it

Reply:
What about it do you not like?

Reply:
The fact that it's so many lines. Having an entire line consist of `1,` seems dumb to me. I'd prefer to have the arguments closer together, but it is something I'm willing to concede

Reply:
I would agree, but there is not much that can be done. We can either pack function args but arrays will also be packed or vice versa. This would mean we can't have arrays that we deliberately layout with a single element on each line. The option is `BinPackArguments` which is currently false.

Reply:
Can't win them all I guess